portfolio/src/main/webapp/script.js

```diff
@@ -72,3 +75,13 @@ function showComments(data) {
   }
   commentContainer.appendChild(arrayToListElement(data));
 }
+
+function processLoginStatus(status) {
+  if (status === "Logged") {
```

Comment:
Let's make this more realistic use case.
If the user is logged in, then he/she can submit a new comment.
Else he/she can only see comments.